lib/canary/plugs.ex

```diff
@@ -37,10 +37,15 @@ defmodule Canary.Plugs do
   If the resource cannot be fetched, `conn.assigns.resource_name` is set
   to nil.
 
-  If the action is `:index`, all records from the specified model will be loaded.
+  The `:persisted` key can override how a resource is loaded and can be useful when dealing
+  with nested resources.
+
+  If the action is `:index`, all records from the specified model will be loaded. This can
```

Comment:
I went ahead and moved the "The :persisted key can override..." sentence below the two paragraphs about the :index, :new, and :create default behaviors.  I also added "By default, when the action is..." to the paragraph about the :index action.